armi/bookkeeping/visualization/xdmf.py

```diff
@@ -287,7 +287,6 @@ def dumpState(
         node = r.p.timeNode
 
         timeGroupName = database3.getH5GroupName(cycle, node)
-        dbVersion = self._inputDb.version
```

Comment:
I checked. There are no side effects in the "version" getter.  There is one in the setter though. So it would make a difference if we were going in the other direction.